apps/api/src/test-data/buildEdgeConnectionTestData/buildDualEdgeConnectionTestData/buildOneDualEdgeConnectionForEveryContextType.ts

```diff
@@ -1,3 +1,134 @@
-import { EdgeConnection } from '../../../domain/models/context/edge-connection.entity';
+import {
+    EdgeConnection,
+    EdgeConnectionMemberRole,
+    EdgeConnectionType,
+} from '../../../domain/models/context/edge-connection.entity';
+import { GeneralContext } from '../../../domain/models/context/general-context/general-context.entity';
+import { PageRangeContext } from '../../../domain/models/context/page-range-context/page-range.context.entity';
+import { EdgeConnectionContextType } from '../../../domain/models/context/types/EdgeConnectionContextType';
+import { AggregateType } from '../../../domain/types/AggregateType';
+import { ResourceType } from '../../../domain/types/ResourceType';
 
-export default (): EdgeConnection[] => [];
+export default (): EdgeConnection[] =>
+    [
+        {
```

Comment:
Might want to check if we can opt out of some more auto-generation of dual connections at this point.